yarn.lock

```diff
@@ -11417,7 +11391,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"react-addons-shallow-compare@npm:^15.6.2, react-addons-shallow-compare@npm:^15.6.3":
+"react-addons-shallow-compare@npm:^15.6.3":
```

Comment:
Evidently this is still required (via `react-nestable`) but it doesn't need to be listed in package.json, `react-nestable` can install it itself.